_builder_lib/docsitebuilder/helpers.rb

```diff
@@ -15,7 +15,19 @@ module Helpers
     PACKAGE_DIRNAME     = '_package'
     BLANK_STRING_RE     = Regexp.new('^\s*$')
     PRODUCT_AUTHOR      = "OpenShift Documentation Project <dev@lists.openshift.redhat.com>"
-    ANALYTICS_SHIM      = '<script type="text/javascript" src="https://assets.openshift.net/app/assets/site/tracking.js"></script>'
+    ANALYTICS_SHIM      = {
+      'OpenShift Origin' => "<script async src=\"//www.google-analytics.com/analytics.js\" type=\"text/javascript\"></script>
```

Comment:
I would prefer to see this hash organized by distro key (`openshift-origin`, `openshift-online`, `openshift-enterprise`). This means that the lookup will need to be done in a slightly different place...